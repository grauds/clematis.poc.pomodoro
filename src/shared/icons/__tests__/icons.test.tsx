import React from "react";
import {shallow} from "enzyme";
import fs from "fs";

const importIcon = async (path: string): Promise<React.JSX.Element> => {
  const module = await import(path);
  const Icon = module.default
  return <Icon />
};

function readIcons(path: string): [string, Promise<React.JSX.Element>][] {
  const absolutePath = `${__dirname}${path}`;
  return fs
    .readdirSync(absolutePath)
    .filter((file) => {
        return file.slice(-4) === ".tsx"
    })
    .map((file) => {
      return [file, importIcon(`${absolutePath}${file}`)]
    });
}

describe("icons", () => {
   test.each(readIcons('/../'))('%s',(_name: string, Icon: Promise<React.JSX.Element>) => {
      Icon.then((Result) => {
        expect(shallow(Result)).toMatchSnapshot();
      })
   });
});
