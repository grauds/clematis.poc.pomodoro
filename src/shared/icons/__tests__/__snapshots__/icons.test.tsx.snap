// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`icons ArrowIcon.tsx 1`] = `
<svg
  fill="none"
  height="10"
  viewBox="0 0 16 10"
  width="16"
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M15 1L8 8L1 1"
    stroke="#B7280F"
    strokeWidth="2"
  />
</svg>
`;

exports[`icons ClockIcon.tsx 1`] = `
<svg
  fill="none"
  height="129"
  viewBox="0 0 129 129"
  width="129"
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M64.3158 118.632C94.3136 118.632 118.632 94.3136 118.632 64.3158C118.632 34.318 94.3136 10 64.3158 10C34.318 10 10 34.318 10 64.3158C10 94.3136 34.318 118.632 64.3158 118.632Z"
    stroke="#C4C4C4"
    strokeLinecap="round"
    strokeLinejoin="round"
    strokeWidth="5"
  />
  <path
    d="M64.3154 37.1579V64.3158L77.8944 77.8947"
    stroke="#C4C4C4"
    strokeLinecap="round"
    strokeLinejoin="round"
    strokeWidth="5"
  />
</svg>
`;

exports[`icons CloseIcon.tsx 1`] = `
<svg
  fill="none"
  height="14"
  viewBox="0 0 15 14"
  width="15"
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M7.9115 8.80583L2.84406 13.9567L0.961665 12.0433L6.02911 6.89245L1.0675 1.84914L2.85992 0.0272091L7.82153 5.07052L12.7673 0.0433371L14.6497 1.95672L9.70392 6.9839L14.6655 12.0272L12.8731 13.8491L7.9115 8.80583Z"
    fill="#C4C4C4"
  />
</svg>
`;

exports[`icons DeleteIcon.tsx 1`] = `
<svg
  fill="none"
  height="18"
  viewBox="0 0 18 18"
  width="18"
  xmlns="http://www.w3.org/2000/svg"
>
  <g
    clipPath="url(#clip0_35_114)"
  >
    <path
      d="M12 6.75V14.25H6V6.75H12ZM10.875 2.25H7.125L6.375 3H3.75V4.5H14.25V3H11.625L10.875 2.25ZM13.5 5.25H4.5V14.25C4.5 15.075 5.175 15.75 6 15.75H12C12.825 15.75 13.5 15.075 13.5 14.25V5.25Z"
      fill="#A8B64F"
    />
  </g>
  <defs>
    <clipPath
      id="clip0_35_114"
    >
      <rect
        fill="white"
        height="18"
        width="18"
      />
    </clipPath>
  </defs>
</svg>
`;

exports[`icons EditIcon.tsx 1`] = `
<svg
  fill="none"
  height="18"
  viewBox="0 0 18 18"
  width="18"
  xmlns="http://www.w3.org/2000/svg"
>
  <g
    clipPath="url(#clip0_35_109)"
  >
    <path
      d="M10.545 6.765L11.235 7.455L4.44 14.25H3.75V13.56L10.545 6.765ZM13.245 2.25C13.0575 2.25 12.8625 2.325 12.72 2.4675L11.3475 3.84L14.16 6.6525L15.5325 5.28C15.825 4.9875 15.825 4.515 15.5325 4.2225L13.7775 2.4675C13.6275 2.3175 13.44 2.25 13.245 2.25ZM10.545 4.6425L2.25 12.9375V15.75H5.0625L13.3575 7.455L10.545 4.6425Z"
      fill="#A8B64F"
    />
  </g>
  <defs>
    <clipPath
      id="clip0_35_109"
    >
      <rect
        fill="white"
        height="18"
        width="18"
      />
    </clipPath>
  </defs>
</svg>
`;

exports[`icons EllipseIcon.tsx 1`] = `
<svg
  fill="none"
  height="25"
  viewBox="0 0 25 25"
  width="25"
  xmlns="http://www.w3.org/2000/svg"
>
  <circle
    cx="12.5"
    cy="12.5"
    r="12"
    stroke="#C4C4C4"
  />
</svg>
`;

exports[`icons HomeIcon.tsx 1`] = `
<svg
  fill="none"
  height="40"
  viewBox="0 0 40 40"
  width="40"
  xmlns="http://www.w3.org/2000/svg"
>
  <g
    clipPath="url(#clip0_23799_226)"
  >
    <path
      d="M38.9151 23.2834C38.9151 33.7058 30.466 40 20.0437 40C9.62098 40 1.17188 31.5509 1.17188 21.1282C1.17188 10.7059 9.88496 4.2981 20.3073 4.2981C30.73 4.2981 38.9151 12.8607 38.9151 23.2834Z"
      fill="#DC3E22"
    />
    <path
      d="M28.238 12.6066C27.3211 11.673 25.8377 10.8048 24.733 10.551C25.3401 10.0127 25.4623 9.99494 26.2227 9.61816C28.1713 8.65365 31.0576 8.56485 31.0576 8.56485C31.0576 8.56485 27.6509 6.8042 25.1601 6.91468C24.5259 6.94257 23.8571 7.16658 23.2118 7.48403C23.5757 6.97054 23.9205 6.45998 24.1409 6.07643C24.8152 4.90368 25.524 3.42627 25.524 3.42627C25.524 3.42627 22.9122 3.56573 21.7008 5.01565C21.2407 5.56645 20.8934 6.26625 20.6392 6.92275C20.1878 6.40419 19.6896 5.94242 19.1913 5.58195C16.6999 3.77896 12.7192 4.16903 12.7192 4.16903C12.7192 4.16903 15.7263 5.87486 17.0793 7.57656C17.6076 8.2411 18.1437 8.54842 18.4642 9.29352C17.3564 9.05367 14.8569 9.13565 13.63 9.59057C10.4771 10.7599 9.11852 15.4649 9.11852 15.4649C9.11852 15.4649 12.1952 13.3443 16.3813 11.8565C17.3017 11.5295 18.2748 11.4429 19.1229 11.4578C18.7379 12.0575 18.3173 12.8363 17.999 13.7546C17.2247 15.9904 18.2479 21.3113 18.2479 21.3113C18.2479 21.3113 20.4896 18.1647 21.403 15.6157C21.8718 14.3073 21.9879 12.9936 21.9904 12.0242C22.8217 12.3931 23.8009 12.9319 24.5326 13.398C28.2794 15.7852 30.072 20.1435 30.072 20.1435C30.072 20.1435 30.5941 15.006 28.238 12.6066Z"
      fill="#899441"
    />
    <path
      d="M20.5008 10.3094C20.4889 10.3094 20.477 10.3091 20.4651 10.3088C19.7242 10.2896 19.1391 9.67376 19.1572 8.9334C19.1587 8.86931 19.2234 4.36125 16.7191 2.40111C16.135 1.94395 16.0318 1.09984 16.489 0.515424C16.9465 -0.0686834 17.7906 -0.171833 18.3747 0.285626C21.9559 3.08806 21.8491 8.76128 21.843 9.00145C21.8238 9.73083 21.2262 10.3094 20.5008 10.3094Z"
      fill="#A8B64F"
    />
  </g>
  <defs>
    <clipPath
      id="clip0_23799_226"
    >
      <rect
        fill="white"
        height="40"
        width="40"
      />
    </clipPath>
  </defs>
</svg>
`;

exports[`icons MenuIcon.tsx 1`] = `
<svg
  fill="none"
  height="20"
  viewBox="0 0 5 20"
  width="5"
  xmlns="http://www.w3.org/2000/svg"
>
  <circle
    cx="2.5"
    cy="2.5"
    fill="#D9D9D9"
    r="2.5"
  />
  <circle
    cx="2.5"
    cy="10"
    fill="#D9D9D9"
    r="2.5"
  />
  <circle
    cx="2.5"
    cy="17.5"
    fill="#D9D9D9"
    r="2.5"
  />
</svg>
`;

exports[`icons MinusIcon.tsx 1`] = `
<svg
  fill="none"
  height="18"
  viewBox="0 0 18 18"
  width="18"
  xmlns="http://www.w3.org/2000/svg"
>
  <g
    clipPath="url(#clip0_35_103)"
  >
    <path
      d="M9 1.5C4.8675 1.5 1.5 4.8675 1.5 9C1.5 13.1325 4.8675 16.5 9 16.5C13.1325 16.5 16.5 13.1325 16.5 9C16.5 4.8675 13.1325 1.5 9 1.5ZM9 15C5.6925 15 3 12.3075 3 9C3 5.6925 5.6925 3 9 3C12.3075 3 15 5.6925 15 9C15 12.3075 12.3075 15 9 15Z"
      fill="#A8B64F"
    />
    <path
      d="M5.25 8.25H8.25H9.75H12.75V9.75H9.75H8.25H5.25V8.25Z"
      fill="#A8B64F"
    />
  </g>
  <defs>
    <clipPath
      id="clip0_35_103"
    >
      <rect
        fill="white"
        height="18"
        width="18"
      />
    </clipPath>
  </defs>
</svg>
`;

exports[`icons PlusBigIcon.tsx 1`] = `
<svg
  fill="none"
  height="50"
  viewBox="0 0 50 50"
  width="50"
  xmlns="http://www.w3.org/2000/svg"
>
  <circle
    cx="25"
    cy="25"
    fill="#C4C4C4"
    r="25"
  />
  <path
    d="M26.2756 26.1321V33H23.7244V26.1321H17V23.7029H23.7244V17H26.2756V23.7029H33V26.1321H26.2756Z"
    fill="white"
  />
</svg>
`;

exports[`icons PlusIcon.tsx 1`] = `
<svg
  fill="none"
  height="18"
  viewBox="0 0 18 18"
  width="18"
  xmlns="http://www.w3.org/2000/svg"
>
  <g
    clipPath="url(#clip0_35_100)"
  >
    <path
      d="M9.75 5.25H8.25V8.25H5.25V9.75H8.25V12.75H9.75V9.75H12.75V8.25H9.75V5.25ZM9 1.5C4.8675 1.5 1.5 4.8675 1.5 9C1.5 13.1325 4.8675 16.5 9 16.5C13.1325 16.5 16.5 13.1325 16.5 9C16.5 4.8675 13.1325 1.5 9 1.5ZM9 15C5.6925 15 3 12.3075 3 9C3 5.6925 5.6925 3 9 3C12.3075 3 15 5.6925 15 9C15 12.3075 12.3075 15 9 15Z"
      fill="#A8B64F"
    />
  </g>
  <defs>
    <clipPath
      id="clip0_35_100"
    >
      <rect
        fill="white"
        height="18"
        width="18"
      />
    </clipPath>
  </defs>
</svg>
`;

exports[`icons PomodoroIcon.tsx 1`] = `
<svg
  fill="none"
  height="115"
  viewBox="0 0 115 115"
  width="115"
  xmlns="http://www.w3.org/2000/svg"
>
  <g
    clipPath="url(#clip0_16_527)"
  >
    <path
      d="M111.881 66.9398C111.881 96.9041 87.5898 115 57.6255 115C27.6603 115 3.36914 90.7089 3.36914 60.7437C3.36914 30.7794 28.4192 12.3571 58.3836 12.3571C88.3488 12.3571 111.881 36.9746 111.881 66.9398Z"
      fill="#DC3E22"
    />
    <path
      d="M81.185 36.2439C78.5489 33.5598 74.284 31.0639 71.1081 30.3341C72.8534 28.7865 73.2046 28.7355 75.391 27.6522C80.9932 24.8793 89.2913 24.624 89.2913 24.624C89.2913 24.624 79.497 19.5621 72.3358 19.8797C70.5127 19.9599 68.5898 20.604 66.7346 21.5166C67.7807 20.0403 68.7719 18.5725 69.4056 17.4698C71.3442 14.0981 73.3822 9.85057 73.3822 9.85057C73.3822 9.85057 65.8733 10.2515 62.3903 14.42C61.0676 16.0036 60.0691 18.0155 59.3384 19.9029C58.0406 18.4121 56.6082 17.0845 55.1756 16.0481C48.0129 10.8645 36.5683 11.986 36.5683 11.986C36.5683 11.986 45.2138 16.8902 49.1036 21.7826C50.6224 23.6932 52.1638 24.5767 53.0852 26.7189C49.9003 26.0293 42.7142 26.265 39.1867 27.5729C30.1222 30.9348 26.2164 44.4617 26.2164 44.4617C26.2164 44.4617 35.0618 38.3648 47.0968 34.0876C49.743 33.1475 52.5406 32.8983 54.9789 32.9411C53.872 34.6654 52.6628 36.9045 51.7478 39.5446C49.5215 45.9724 52.4634 61.2701 52.4634 61.2701C52.4634 61.2701 58.9082 52.2234 61.5343 44.8951C62.8821 41.1335 63.2157 37.3568 63.2231 34.5698C65.6131 35.6302 68.4281 37.1793 70.5319 38.5193C81.3038 45.3826 86.4576 57.9127 86.4576 57.9127C86.4576 57.9127 87.9585 43.1422 81.185 36.2439Z"
      fill="#899441"
    />
    <path
      d="M58.9395 29.6396C58.9053 29.6396 58.8711 29.6388 58.8368 29.6379C56.7066 29.5826 55.0246 27.8121 55.0764 25.6835C55.0808 25.4993 55.2668 12.5386 48.067 6.90318C46.3877 5.58886 46.0911 3.16203 47.4054 1.48184C48.7206 -0.197465 51.1475 -0.49402 52.8268 0.821175C63.1229 8.87817 62.8158 25.1887 62.7982 25.8792C62.7429 27.9761 61.025 29.6396 58.9395 29.6396Z"
      fill="#A8B64F"
    />
    <circle
      cx="41.5"
      cy="64.5"
      fill="black"
      r="2.5"
    />
    <g
      filter="url(#filter0_f_16_527)"
    >
      <circle
        cx="29.5"
        cy="75.5"
        fill="#EA8979"
        r="5.5"
      />
    </g>
    <g
      filter="url(#filter1_f_16_527)"
    >
      <circle
        cx="85.5"
        cy="75.5"
        fill="#EA8979"
        r="5.5"
      />
    </g>
    <circle
      cx="73.5"
      cy="64.5"
      fill="black"
      r="2.5"
    />
    <path
      d="M46 78C50 82 64.5 83 68.5 78"
      stroke="black"
    />
  </g>
  <defs>
    <filter
      colorInterpolationFilters="sRGB"
      filterUnits="userSpaceOnUse"
      height="19"
      id="filter0_f_16_527"
      width="19"
      x="20"
      y="66"
    >
      <feFlood
        floodOpacity="0"
        result="BackgroundImageFix"
      />
      <feBlend
        in="SourceGraphic"
        in2="BackgroundImageFix"
        mode="normal"
        result="shape"
      />
      <feGaussianBlur
        result="effect1_foregroundBlur_16_527"
        stdDeviation="2"
      />
    </filter>
    <filter
      colorInterpolationFilters="sRGB"
      filterUnits="userSpaceOnUse"
      height="19"
      id="filter1_f_16_527"
      width="19"
      x="76"
      y="66"
    >
      <feFlood
        floodOpacity="0"
        result="BackgroundImageFix"
      />
      <feBlend
        in="SourceGraphic"
        in2="BackgroundImageFix"
        mode="normal"
        result="shape"
      />
      <feGaussianBlur
        result="effect1_foregroundBlur_16_527"
        stdDeviation="2"
      />
    </filter>
    <clipPath
      id="clip0_16_527"
    >
      <rect
        fill="white"
        height="115"
        width="115"
      />
    </clipPath>
  </defs>
</svg>
`;

exports[`icons PomodoroSmallIcon.tsx 1`] = `
<svg
  fill="none"
  height="81"
  viewBox="0 0 81 81"
  width="81"
  xmlns="http://www.w3.org/2000/svg"
>
  <g
    clipPath="url(#clip0_16_413)"
  >
    <path
      d="M78.8031 47.4452C78.8031 68.5505 61.6937 81.2964 40.5884 81.2964C19.4825 81.2964 2.37305 64.187 2.37305 43.081C2.37305 21.9757 20.017 9 41.1223 9C62.2283 9 78.8031 26.3393 78.8031 47.4452Z"
      fill="#DC3E22"
    />
    <path
      d="M57.1821 25.5283C55.3254 23.6378 52.3214 21.8798 50.0845 21.3658C51.3138 20.2757 51.5612 20.2398 53.1011 19.4768C57.0471 17.5237 62.8917 17.3439 62.8917 17.3439C62.8917 17.3439 55.9932 13.7785 50.9492 14.0023C49.6651 14.0587 48.3107 14.5124 47.004 15.1552C47.7408 14.1154 48.439 13.0815 48.8853 12.3048C50.2508 9.92999 51.6862 6.93824 51.6862 6.93824C51.6862 6.93824 46.3974 7.22064 43.9441 10.1567C43.0125 11.2721 42.3092 12.6892 41.7945 14.0186C40.8804 12.9685 39.8715 12.0334 38.8625 11.3035C33.8174 7.65243 25.7565 8.44232 25.7565 8.44232C25.7565 8.44232 31.8459 11.8966 34.5857 15.3426C35.6554 16.6883 36.7411 17.3106 37.3901 18.8194C35.1468 18.3337 30.0853 18.4997 27.6007 19.4209C21.2162 21.7889 18.4651 31.3165 18.4651 31.3165C18.4651 31.3165 24.6953 27.0222 33.1722 24.0095C35.036 23.3474 37.0065 23.1719 38.7239 23.202C37.9443 24.4165 37.0926 25.9936 36.4481 27.8532C34.88 32.3806 36.9521 43.1555 36.9521 43.1555C36.9521 43.1555 41.4915 36.7835 43.3412 31.6218C44.2905 28.9723 44.5255 26.3122 44.5307 24.3491C46.2141 25.0961 48.1968 26.1872 49.6786 27.131C57.2658 31.9651 60.8959 40.7907 60.8959 40.7907C60.8959 40.7907 61.9531 30.3871 57.1821 25.5283Z"
      fill="#899441"
    />
    <path
      d="M41.5144 20.8766C41.4903 20.8766 41.4662 20.876 41.4421 20.8754C39.9416 20.8364 38.757 19.5894 38.7934 18.0901C38.7965 17.9604 38.9275 8.83154 33.8564 4.86224C32.6735 3.9365 32.4647 2.22717 33.3904 1.04373C34.3168 -0.139084 36.0261 -0.347962 37.2089 0.578393C44.4609 6.25332 44.2446 17.7416 44.2323 18.2279C44.1933 19.7049 42.9833 20.8766 41.5144 20.8766Z"
      fill="#A8B64F"
    />
  </g>
  <defs>
    <clipPath
      id="clip0_16_413"
    >
      <rect
        fill="white"
        height="81"
        width="81"
      />
    </clipPath>
  </defs>
</svg>
`;

exports[`icons SettingsIcon.tsx 1`] = `
<svg
  fill="none"
  height="20"
  viewBox="0 0 20 20"
  width="20"
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M16.1915 10.8165C16.2249 10.5499 16.2499 10.2832 16.2499 9.99988C16.2499 9.71654 16.2249 9.44988 16.1915 9.18321L17.9499 7.80821C18.1082 7.68321 18.1499 7.45821 18.0499 7.27488L16.3832 4.39154C16.3341 4.30556 16.2559 4.23989 16.1628 4.2063C16.0696 4.1727 15.9676 4.17338 15.8749 4.20821L13.7999 5.04154C13.3665 4.70821 12.8999 4.43321 12.3915 4.22488L12.0749 2.01654C12.0611 1.91849 12.012 1.82884 11.9368 1.76441C11.8616 1.69997 11.7655 1.66517 11.6665 1.66654H8.3332C8.12487 1.66654 7.94987 1.81654 7.92487 2.01654L7.6082 4.22488C7.09987 4.43321 6.6332 4.71654 6.19987 5.04154L4.12487 4.20821C4.07655 4.19182 4.02589 4.18337 3.97487 4.18321C3.8332 4.18321 3.69153 4.25821 3.61653 4.39154L1.94987 7.27488C1.84153 7.45821 1.89153 7.68321 2.04987 7.80821L3.8082 9.18321C3.77487 9.44988 3.74987 9.72488 3.74987 9.99988C3.74987 10.2749 3.77487 10.5499 3.8082 10.8165L2.04987 12.1915C1.89153 12.3165 1.84987 12.5415 1.94987 12.7249L3.61653 15.6082C3.66566 15.6942 3.7438 15.7599 3.83695 15.7935C3.9301 15.8271 4.03217 15.8264 4.12487 15.7915L6.19987 14.9582C6.6332 15.2915 7.09987 15.5665 7.6082 15.7749L7.92487 17.9832C7.94987 18.1832 8.12487 18.3332 8.3332 18.3332H11.6665C11.8749 18.3332 12.0499 18.1832 12.0749 17.9832L12.3915 15.7749C12.8999 15.5665 13.3665 15.2832 13.7999 14.9582L15.8749 15.7915C15.9249 15.8082 15.9749 15.8165 16.0249 15.8165C16.1665 15.8165 16.3082 15.7415 16.3832 15.6082L18.0499 12.7249C18.1499 12.5415 18.1082 12.3165 17.9499 12.1915L16.1915 10.8165V10.8165ZM14.5415 9.39154C14.5749 9.64988 14.5832 9.82488 14.5832 9.99988C14.5832 10.1749 14.5665 10.3582 14.5415 10.6082L14.4249 11.5499L15.1665 12.1332L16.0665 12.8332L15.4832 13.8415L14.4249 13.4165L13.5582 13.0665L12.8082 13.6332C12.4499 13.8999 12.1082 14.0999 11.7665 14.2415L10.8832 14.5999L10.7499 15.5415L10.5832 16.6665H9.41653L9.2582 15.5415L9.12487 14.5999L8.24153 14.2415C7.8832 14.0915 7.54987 13.8999 7.21653 13.6499L6.4582 13.0665L5.57487 13.4249L4.51653 13.8499L3.9332 12.8415L4.8332 12.1415L5.57487 11.5582L5.4582 10.6165C5.4332 10.3582 5.41653 10.1665 5.41653 9.99988C5.41653 9.83321 5.4332 9.64154 5.4582 9.39154L5.57487 8.44988L4.8332 7.86654L3.9332 7.16654L4.51653 6.15821L5.57487 6.58321L6.44153 6.93321L7.19153 6.36654C7.54987 6.09988 7.89153 5.89988 8.2332 5.75821L9.11653 5.39988L9.24987 4.45821L9.41653 3.33321H10.5749L10.7332 4.45821L10.8665 5.39988L11.7499 5.75821C12.1082 5.90821 12.4415 6.09988 12.7749 6.34988L13.5332 6.93321L14.4165 6.57488L15.4749 6.14988L16.0582 7.15821L15.1665 7.86654L14.4249 8.44988L14.5415 9.39154ZM9.99987 6.66654C8.1582 6.66654 6.66653 8.15821 6.66653 9.99988C6.66653 11.8415 8.1582 13.3332 9.99987 13.3332C11.8415 13.3332 13.3332 11.8415 13.3332 9.99988C13.3332 8.15821 11.8415 6.66654 9.99987 6.66654ZM9.99987 11.6665C9.0832 11.6665 8.3332 10.9165 8.3332 9.99988C8.3332 9.08321 9.0832 8.33321 9.99987 8.33321C10.9165 8.33321 11.6665 9.08321 11.6665 9.99988C11.6665 10.9165 10.9165 11.6665 9.99987 11.6665Z"
    fill="#DC3E22"
  />
</svg>
`;

exports[`icons StatsIcon.tsx 1`] = `
<svg
  fill="none"
  height="24"
  viewBox="0 0 24 24"
  width="24"
  xmlns="http://www.w3.org/2000/svg"
>
  <g
    clipPath="url(#clip0_23799_232)"
  >
    <path
      d="M10 20H14V4H10V20ZM4 20H8V12H4V20ZM16 9V20H20V9H16Z"
      fill="#DC3E22"
    />
  </g>
  <defs>
    <clipPath
      id="clip0_23799_232"
    >
      <rect
        fill="white"
        height="24"
        width="24"
      />
    </clipPath>
  </defs>
</svg>
`;

exports[`icons StopIcon.tsx 1`] = `
<svg
  fill="none"
  height="129"
  viewBox="0 0 129 129"
  width="129"
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M64.3158 118.632C94.3136 118.632 118.632 94.3136 118.632 64.3158C118.632 34.318 94.3136 10 64.3158 10C34.318 10 10 34.318 10 64.3158C10 94.3136 34.318 118.632 64.3158 118.632Z"
    stroke="#C4C4C4"
    strokeLinecap="round"
    strokeLinejoin="round"
    strokeWidth="5"
  />
  <path
    d="M28 27L102 101"
    stroke="#C4C4C4"
    strokeLinecap="round"
    strokeLinejoin="round"
    strokeWidth="5"
  />
</svg>
`;

exports[`icons TargetIcon.tsx 1`] = `
<svg
  fill="none"
  height="129"
  viewBox="0 0 129 129"
  width="129"
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M64.3158 118.632C94.3136 118.632 118.632 94.3136 118.632 64.3158C118.632 34.318 94.3136 10 64.3158 10C34.318 10 10 34.318 10 64.3158C10 94.3136 34.318 118.632 64.3158 118.632Z"
    stroke="#C4C4C4"
    strokeLinecap="round"
    strokeLinejoin="round"
    strokeWidth="5"
  />
  <path
    d="M64.5 102C85.2107 102 102 85.2107 102 64.5C102 43.7893 85.2107 27 64.5 27C43.7893 27 27 43.7893 27 64.5C27 85.2107 43.7893 102 64.5 102Z"
    stroke="#C4C4C4"
    strokeLinecap="round"
    strokeLinejoin="round"
    strokeWidth="5"
  />
  <path
    d="M64.5 85C75.8218 85 85 75.8218 85 64.5C85 53.1782 75.8218 44 64.5 44C53.1782 44 44 53.1782 44 64.5C44 75.8218 53.1782 85 64.5 85Z"
    stroke="#C4C4C4"
    strokeLinecap="round"
    strokeLinejoin="round"
    strokeWidth="5"
  />
</svg>
`;
